api/v1alpha1/managedfleetnotificationrecord_types.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+// ManagedFleetNotificationRecordStatus defines the observed state of ManagedFleetNotificationRecord
+type ManagedFleetNotificationRecordStatus struct {
+	// Managed Fleet Notification name
+	ManagementCluster string `json:"managementCluster"`
+	// Resend interval for the notification
+	ResendWait int32 `json:"resendWait"`
```

Comment:
I don't really get the `ResendWait` here. This is going to be potentially different per notification, but its existence at a broad level implies 'one resend value for all'.

Reply:
ah, you are right, I think it should be put the same level with the notification name

Reply:
set the `ResendWait` as per notification

Reply:
Thanks for the change, that fits better